tests/test_anno.py

```diff
@@ -69,3 +69,27 @@ def test_anno(self) -> None:
             x-feature  Custom annotation for `feature` branch (untracked)
             """,
         )
+
+        # check if annotation qualifiers are parsed correctly and that they can be overwritten by `git machete anno`
+        launch_command('anno', '-b=refs/heads/feature', 'push=no Custom annotation for `feature` branch rebase=no')
+        assert_command(
+            ["status"],
+            """
+            master (untracked)
+
+            develop *  Custom annotation for `develop` branch (untracked)
+            |
+            x-feature  Custom annotation for `feature` branch rebase=no push=no (untracked)
```

Comment:
TBD if this reordering (all qualifiers go to the end) is needed... albeit if that's easier to implement, it can stay ofc

Reply:
Well, it's not easier to implement but it makes them easier to find in the annotation, also it's not required, you can just put them anywhere you want, it will just get formatted this way so it requires no extra work.